src/components/TipInput.vue

```diff
@@ -121,6 +123,25 @@ export default {
         };
       },
     }),
+    largestFtTipAmount() {
+      return this.tipUrlStats.tokenTotalAmount.length
+        ? [...this.tipUrlStats.tokenTotalAmount].sort((a, b) => (
+          shiftDecimalPlaces(new BigNumber(b.amount), -this.tokenInfo[b.token].decimals)
```

Comment:
`shiftDecimalPlaces` [already](https://github.com/aeternity/superhero-ui/blob/32076e07b0ac1ad249be984995546e2505fc7890/src/utils/index.js#L7) does the conversion to BigNumber, use one of:
```suggestion
          new BigNumber(b.amount).shiftedBy(-this.tokenInfo[b.token].decimals)
```
or
```suggestion
          shiftDecimalPlaces(b.amount, -this.tokenInfo[b.token].decimals)
```